controllers/node/controllers/gpu_controller.go

```diff
@@ -153,6 +167,9 @@ func (r *GpuReconciler) Reconcile(ctx context.Context, _ ctrl.Request) (ctrl.Res
 			return ctrl.Result{}, err
 		}
 	} else if err == nil {
+		if configmap.Data == nil {
```

Comment:
use err != nil or just delete `else if`

Reply:
is it ok?
![image](https://github.com/labring/sealos/assets/72536832/9e05a198-09f5-4d7b-9ce8-ae3cbfe27c20)